Integrations/integration-Forcepoint.yml

```diff
@@ -0,0 +1,573 @@
+description: Advanced threat protection with added local management controls.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: User name
```

Comment:
Just a minor comment, if it holds both username and password display should be credentials and not User name